packages/apps/dashboard/server/src/common/config/env-config.service.ts

```diff
@@ -93,10 +93,10 @@ export class EnvironmentConfigService {
     );
   }
 
-  get networkAvailableCacheTtl(): number {
+  get networkOperatingCacheTtl(): number {
     return this.configService.get<number>(
```

Comment:
We should multiple the result by `1000` to return value in milliseconds as it's expected by `cacheManager`

Also need to add `NETWORKS_OPERATING_CACHE_TTL` to `validationSchema` when initializing config, so it's actually a `number`, not `string` when read from env